### ISSUE TITLE
Follow the community-driven Ruby coding style guide

### ISSUE THREAD

**Issue by ghost** [rrrene/sparkr#6]
Hey, I was wondering whether you'd be interest in a Pull Request that fixes offences shown by [Rubocop](https://github.com/bbatsov/rubocop) when ran on your project, thus making the project follow the [community-driven Ruby coding style guide](https://github.com/bbatsov/ruby-style-guide). The fixes are mostly very small and consider things like single over double quotes, whitespace, as well as spaces between `,`'s (and a little more). I think it's a nice touch to follow some coding conventions in a project so the code is consistent and future contributors don't need to worry about looking through the whole code base in order to grasp its coding style.

I've ran `rubocop -a` to automatically fix about 71 out of 82 total offences, and [committed them to my fork of this project](https://github.com/rafalchmiel/sparkr/commit/4175442698ee1f888b3dd025df945909faee7c80) (you can see the exact changes very clearly). There are 11 more to fix but they're very small (like exceeding the 80 character line length rule and having methods that are considered to be "too long"). _By the way the tests pass with my changes._

**Comment by rrrene** [comment]
In a way, this is a tough one.

I agree with you that following some kind of consistent coding style is beneficial to all present and future users of a project. For example, when I contributed heavily to the terminal-table gem in 2011, I followed their style although it wasn't my cup of tea, just so my contributions would blend in with the rest of the project.

Now to the issue at hand: Rubocop suggests some things that I simply would not do if I had any choice in it. Documenting every module/class, and always using the if-modifier for single line conditional blocks are two examples I got when running it on sparkr. Using only ASCII characters would also be a problem considering the `▁▂█` examples needed in comments.

But you raised some good points: using $LOAD_PATH and inserting spaces after commas, as well as lines exceeding the 80 (or re:git 78)-character limit. I fixed those in the last commit and - since there is no pull request - did the next best thing by attributing the changes to you.

Thanks for raising the issue! :+1:
